### ISSUE TITLE
Monitor Reordering

### ISSUE THREAD

**Issue by sovalyeler** [xanderfrangos/twinkle-tray#390]
Hi. Thank you for this app. Using it with f.lux for months now. Being able to change brightness of individual monitors is really great. Also adding to ability change contrast was a nice addition. However I have an issue. After updating to v1.14.2, I can't reorder monitors. After reordering, monitors disappear from settings menu for a second and they revert back to default order. So I reverted back to v1.14.1. Can you look into this?

**Comment by xanderfrangos** [comment]
Yep, that's a bug. I'll get that fixed soon. 👍 Thanks for the bug report!

**Comment by xanderfrangos** [comment]
Whoops

**Comment by xanderfrangos** [comment]
Okay, there we go. Didn't mean to close the issue. 😅

I've released v1.14.3, which should resolve the issue. Let me know if you're still seeing it. Thanks!
